manifests/instance.pp

```diff
@@ -309,8 +313,8 @@
   }
 
   if $manage_service_file {
-    file { "/etc/systemd/system/${service_name}.service":
-      ensure  => file,
+    systemd::unit_file { "${service_name}.service":
```

Comment:
If you pass enable and active as well, the unit file can manage the service and also properly apply `daemon_reload`. This is particularly important now that `camptocamp/systemd` has dropped the `daemon_reload` class in master.

Reply:
See https://github.com/voxpupuli/puppet-redis/commit/58aa20ccded225c5acab3cd2ad24f6e595f6632a for what I think is roughly needed.